system-variables.md

```diff
@@ -427,6 +428,17 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - Default value: ON
 - This variable controls whether to record the execution information of each operator in the slow query log.
 
+### tidb_enable_enhanced_security
+
+- Scope: NONE
+- Default value: OFF
+- This variable indicates if the TiDB server you are connected to has Security Enhanced Mode (SEM) enabled, and can not be changed without restarting the TiDB server.
```

Comment:
@morgo Can we say "To change its value, you need to restart the TiDB server for the change to be effective"?